common/git-hash/src/lib.rs

```diff
@@ -31,13 +31,44 @@ pub fn git_hash_short(_: TokenStream) -> TokenStream {
 }
 
 #[proc_macro]
-pub fn crate_version_with_git_hash(_: TokenStream) -> TokenStream {
```

Comment:
Plz return removed fn `crate_version_with_git_hash `.

Reply:
@fzzr- Thanks. Fixed it